localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,187 @@
  innovation_taming_the_skies:0 "Taming The Skies"
  innovation_taming_the_skies_desc:0 "Our tamers and breeders have been hard at work to successfully allow us to use these flying mounts in battle."
 
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ innovation_archmage:0 "Archmages"
+ innovation_archmage_desc:0 "Weaving their intricate enchantments, these mages defend their people with all the magical powers at their disposal. The archmagi brandish magical blades and ancient staves that serve to channel their fierce energies in battle."
+
+ innovation_buccaneer:0 "Buccaneers"
+ innovation_buccaneer_desc:0 "The buccaneers are the unrivaled lords of the seas. They are highly skilled in sea combat. Their expertise in firearms and sailing aggrandizes their status as the scourge of the seas."
+    
+ innovation_brute:0 "Brutes"
+ innovation_brute_desc:0 "Gnoll brutes are the largest and nastiest gnolls. They wear the best armor and use shields plucked from the bodies of their victims. Scraps of armor cover their thick yellow-green fur, and a wicked looking flail is held in one hand."
+    
+ innovation_candleholder:0 "Candleholders"
+ innovation_candleholder_desc:0 "You no take candle! We protect the Candle King."
+ 
+ innovation_dervishes:0 "Dervishes"
+ innovation_dervishes_desc:0 "The Wastewander Dervishes are the top fighters in the bandit ranks. Their scimitars are so sharp that they cut the wind around them."
+    
+ innovation_dire_troll:0 "Dire Trolls"
+ innovation_dire_troll_desc:0 "These trolls are warriors who have perfected their arts, traded portions of their lives for the dark gift of brutal strength. Dire trolls are short lived but make their presence on this world known to all."
```

The comma in the first sentence feels a bit off. There needs to be something connecting the two parts. I suggested one possible replacement, but feel free to alter it. Also added a dash between short and lived.
```suggestion
 innovation_dire_troll_desc:0 "These trolls are warriors who have perfected their arts before trading portions of their lives in exchange for the dark gift of brutal strength. Dire trolls are short-lived but make their presence on this world known to all."
```